glotaran/project/project.py

```diff
@@ -659,6 +681,7 @@ def optimize(
         result_name: str | None = None,
         maximum_number_function_evaluations: int | None = None,
         clp_link_tolerance: float = 0.0,
+        data_lookup_overwrite: Mapping[str, LoadableDataset] | None = None,
```

Comment:
```suggestion
        data_lookup_override: Mapping[str, LoadableDataset] | None = None,
```

Reply:
Fixed in [5e7a47d](https://github.com/glotaran/pyglotaran/pull/1257/commits/5e7a47de52900b22acde581c9e934280d5e95975)